schema_editor/app/scripts/views/geography/add-partial.html

```diff
@@ -56,7 +56,7 @@ <h2>Add New Geography</h2>
                 </select>
             </div>
         </form>
-        <div class="save-area">
+        <div class="save-area col-sm-12">
```

Comment:
Without assigned column widths, these buttons didn't appear for me in either Chrome or Firefox.

Reply:
Haha, nice, they actually do show up but they're waaaay off to the right. Thanks for fixing this!
![screenshot-2018-6-8 ashlar editor](https://user-images.githubusercontent.com/447977/41156763-43306b8a-6af1-11e8-8cb5-0a1d88297b5c.png)

Reply:
That's where they went! My screen must be too small to see them.